dbtools/pg.py

```diff
@@ -732,6 +736,18 @@ def get_non_geo_columns(self, table: str, schema: str, geometry_col: str = "geom
 
         return columns
 
+    def get_table_column_details(self, table: str, schema: str) -> List[ColumnDetails]:
+        columns_sql = sql.SQL(
+            "SELECT column_name, data_type, is_nullable, character_maximum_length, numeric_precision "
+            "FROM information_schema.columns "
+            "WHERE table_schema = {} AND table_name = {};"
+        ).format(sql.Literal(schema), sql.Literal(table))

```

Comment:
Same comment as above, just want to confirm use of `sql.Literal()` vs `sql.Identifier()`.